### ISSUE TITLE
RIFE 4.25 Fine Tunning

### ISSUE THREAD

**Issue by Farah-mohamed** [hzwer/Practical-RIFE#150]
I am fine tunning RIFE on MODD dataset (water surface movement videos) in order for rife to be better in interpolating videos that contain waves in sea.
I need to know:
a) the fps of the dataset it was trained on, number of images in each dataset and number of epochs it was trained for?
b)when fine tunning which batch size should i choose?
c)i will use the model for sea view 4x interpolation so is it worthy to update the model in each training loop for t=0.25, 0.5 and 0.75 with their gt images, or keep training on a random t selected? if so then what should be the minimum fps of the training dataset to do so? and how much frames per each video sequence?
d)for the learning rate scheduler, can you conclude and equation to adjust its  parameters based on the number of images in the dataset and epochs the model will be  trained on?
Thanks in advance.

**Comment by hzwer** [comment]
a) A suitable training frame rate for RIFE is between 30-60 fps. It is recommended to have over 50,000 images, and generally, more epochs are better.

b) Choose the maximum batch size that your GPU can handle.

c) You can train solely on fixed t values of 0.25, 0.5, and 0.75.

d) Use a cosine decay for the learning rate, meaning a larger learning rate is used for most of the training period, with a rapid decay only at the end.
